### ISSUE TITLE
False dependency on level_zero

### ISSUE THREAD

**Issue by johguenther** [ispc/ispc#2541]
With the split of `ispcrt` in v1.20 `level_zero` is only a runtime dependency when using the GPU device, and not a compile-time dependency when _using_ `ispcrt` (only when building it). Yet, it is required in it's CMake config: https://github.com/ispc/ispc/blob/main/ispcrt/cmake/ispcrtConfig.cmake.in#L27-L38 those line should be removed.

**Comment by nurmukhametov** [comment]
Yes, `level-zero` is not a compile-time dependency for applications linking against `ispcrt`. But was it ever? 
Anyway, it should be fixed. The patch is here https://github.com/ispc/ispc/pull/2543

**Comment by nurmukhametov** [comment]
[Fix](https://github.com/ispc/ispc/commit/89c2fd9eb78168d4d9c1d984219b713e597d1c40) has been propogated
